### PR TITLE
Add Portuguese translation

### PR DESCRIPTION
**Pull request by RuiNascimento**

### CODE REVIEW

**Comment by Blaisorblade** [comment]
Thanks @RuiNascimento! Have you tested your translation already?

**Comment by oskomonzon** [comment]
Looks good. It's missing the latest merged strings (new UI related ones).

**Comment by RuiNascimento** [comment]
Dind't test it, but it seems to be pretty straightforward.
If you could compile the apk for me i can test it.

**Comment by oskomonzon** [comment]
Also, it has two closing tags.

**Comment by RuiNascimento** [comment]
You're right, need to remove one of the closing tags

File: app/src/main/res/values-pt/strings.xml [hunk 1]
```diff
@@ -162,4 +162,57 @@
...
+    <string name="ivtext_no_possibilities">Não há possibilides, por favor verifique os dados!</string>
+    <string name="ivtext_iv">Min: %1$s%% Média: %2$s%% Max: %3$s%%\n</string>
+    <string name="ivtext_evolve">Pode evoluir para %1$s:\n</string>
+    <string name="ivtext_evolve_further">Pode ser ainda mais evoluído para %1$s:</string>
+    <string name="ivtext_many_possibilities">Há muitas possibilidades para este Pokémon. Tente usar o POWER UP!</string>
+    <string name="ivtext_max_lvl_cost">Custo para chegar ao nível %1$s:</string>
+    <string name="ivtext_max_lvl_cost2">Candy: %1$s Stardust: %2$s</string>
+    <string name="main_battery_saver">Poupança de bateria (Pesquiza por captura de ecrã)</string>
+    <string name="ivtext_cp_lvl">CP ao nivél %1$s: %2$s - %3$s</string>
+    <string name="ivtext_stats">Atk: %1$02d Def: %2$02d Sta: %3$02d — %4$3d%%</string>
```
Comment (Blaisorblade) [inline app/src/main/res/values-pt/strings.xml]: Better can drop the ones that are unchanged, so we inherit the main string, OK?

Comment (RuiNascimento) [inline app/src/main/res/values-pt/strings.xml]: Ok

**Comment by Blaisorblade** [comment]
It also seems to use escapes correctly. Well, I guess we can merge it then...

**Comment by Blaisorblade** [comment]
But I'll wait for your latest changes, OK?

**Comment by RuiNascimento** [comment]
Sorry, but what changes?

File: app/src/main/res/values-pt/strings.xml (hunk 1 above)
Comment (Blaisorblade) [inline app/src/main/res/values-pt/strings.xml]: I meant—can you drop these strings, like `ivtext_stats`, that are unchanged compared to the original translation?

**Comment by Blaisorblade** [comment]
I merged anyway, thanks a lot! @DJCrapsody is taking care of some refinements.

**Pull request merged**